src/session.ts

```diff
@@ -52,6 +49,15 @@ export const enum types {
   ReadWrite = 'readwrite',
 }
 
+export type GetSessionMetadataCallback = ResourceCallback<
+  spanner_client.spanner.v1.ISession,
+  r.Response
+>;
+type GetSessionMetadataResponse = [
+  spanner_client.spanner.v1.ISession | null,
```

Comment:
No need `| null` in response type declarations
```suggestion
  spanner_client.spanner.v1.ISession
```